Assets/Scripts/Interactables/Interactable.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Interactable : MonoBehaviour
+{
+    [Tooltip("The radius around the object that stops character going inside it")]
+    [SerializeField] public float radius = 3f;
```

Comment:
I have to define it as public so i can access the value from PlayerController.

Reply:
If that's the case, you don't have to add the [SerializeField] attribute. But since I guess the player controller shouldn't be able to change the radius, it would be even better if you created a getter for the field and make it private:
```c#
public float Radius => radius;
```